msl/examples/equipment/vaisala/ptb330.py

```diff
@@ -0,0 +1,59 @@
+"""
+Example showing how to communicate with a Vaisala Barometer which reads pressure
+e.g. of type PTB330
+"""
+from msl.equipment import (
+    EquipmentRecord,
+    ConnectionRecord,
+    Backend,
+)
+from msl.equipment.constants import (
+    Parity,
+    DataBits,
+)
+
+import logging
+logging.basicConfig(level=logging.INFO)
+
+record = EquipmentRecord(
+    manufacturer='Vaisala',
+    model='PTB330',
+    serial='L2310094',
+    connection=ConnectionRecord(
+        address='COM9',  # change for your device
+        backend=Backend.MSL,
+        baud_rate=4800,  # change for your device
+        parity=Parity.EVEN,  # change for your device
+        termination='\r\n',
+        timeout=5,
+        data_bits=DataBits.SEVEN,
```

Comment:
Since these are the default RS232 settings, it may be best to define them as the default values in the class and allow the user to overwrite the default values should they need to.

For example,
https://github.com/MSLNZ/msl-equipment/blob/62a8d2ee0eadadf777d1a1c600169effac4d8924/msl/equipment/resources/greisinger/gmh3000.py#L48